pyuvdata/uvdata/mir_parser.py

```diff
@@ -861,37 +1064,47 @@ def _read_data(self, data_type, return_vis=True, use_mmap=True, read_only=False)
         Returns
         -------
         data_dict : dict
-            A dictionary, whose the keys are matched to individual values of sphid in
-            `sp_data`, and each entry contains a dict with two items. If
-            `return_vis=False` then a "raw data" dict is passed, with keys "scale_fac",
-            an np.int16 which describes the common exponent for the spectrum, and
-            "data", an array of np.int16 (of length equal to twice that found in
-            `sp_data["nch"]` for the corresponding value of sphid) containing the
-            compressed visibilities.  Note that entries equal to -32768 aren't possible
-            with the compression scheme used for MIR, and so this value is used to mark
-            flags. If `return_vis=True`, then a "vis data" dict is passed, with keys
-            "data", an array of np.complex64 containing the visibilities, and
-            "flags", an array of bool containing the per-channel flags of the
-            spectrum (both are of length equal to `sp_data["nch"]` for the
-            corresponding value of sphid).
+            A dictionary, whose the keys are matched to individual header keys -- if
```

Comment:
```suggestion
            A dictionary, whose keys are matched to individual header keys -- if
```

Reply:
Fixed in be15abab6f4548a576351a1f43d190cefd2fcd48.